FizzBuzzApp/Program.cs

```diff
@@ -1,3 +1,24 @@
 ﻿// See https://aka.ms/new-console-template for more information
 
-Console.WriteLine("Hello, World!");
\ No newline at end of file
+for (int i = 1; i < 101; i++) {
+    string answer = "";

```

Comment:
Minor but interesting: In CSharp there exists something called a ["StringBuilder"](https://docs.microsoft.com/en-us/dotnet/api/system.text.stringbuilder?view=netcore-3.1) which is slightly more performant when you're amending/appending strings immediately which you could use instead.

Fun fact: This was the comment I got given on my PR when I first did this exercise!